Translation/Feat: Understanding angular>Components>Overview

HRSArgyropoulos opened the pull request:
**Summary**

- Translation of [Understanding angular>Components>Overview](https://angular-gr.web.app/guide/component-overview) page.
- Taxonomies translation table proposal

**Description**

As I was translating this topic/page and comparing it with the other PRs, it became more and more evident that a translation table for some terms might help in keeping the translation of this documentation consistent across all pages (as previously proposed by another member [lleenna](https://github.com/lleenna).
I have created a first implementation of this in the taxonomies-translation.md, thus let me know what you think about it (if find this necessary as well or you have any other suggestions). Thanks! 😃

bampakoa commented:
Καλησπέρα @HRSArgyropoulos 👋 Έχω μεταφέρει τον πολύ χρήσιμο πίνακά σου [εδώ](https://github.com/bampakoa/angular-gr/wiki/%CE%95%CE%BB%CE%BB%CE%B7%CE%BD%CE%B9%CE%BA%CF%8C-%CE%B3%CE%BB%CF%89%CF%83%CF%83%CE%AC%CF%81%CE%B9-%CF%8C%CF%81%CF%89%CE%BD). 

Πολύ καλη η ιδέα σου, σε ευχαριστώ πολύ 🙏

HRSArgyropoulos commented:
- Removed taxonomy markdown table from commit (as it got moved in the Wiki section of this repo)
- Updated the translation of this proposed PR topic to follow the new taxonomies translation from the wiki section.

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
-* A CSS selector that defines how the component is used in a template
-* Optionally, CSS styles applied to the template
+* Ένα HTML template που δηλώνει τι απεικονίζεται στη σελίδα
+* Μια κλάση TypeScript που ορίζει τη συμπεριφορά
```
```suggestion
* Ένα TypeScript class που ορίζει τη συμπεριφορά
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
 </div>
 
-## Prerequisites
+## Προϋποθέσεις
```
```suggestion
## Προαπαιτούμενα
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
+### Δημιουργώντας ενα component, χρησιμοποιώντας το Angular CLI
 
-To create a component using the Angular CLI:
+Για να δημιουργήσετε ένα στοιχείο χρησιμοποιώντας το Angular CLI:
```
```suggestion
Για να δημιουργήσετε ένα component χρησιμοποιώντας το Angular CLI:
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
 
-1. From a terminal window, navigate to the directory containing your application.
-1. Run the `ng generate component <component-name>` command, where `<component-name>` is the name of your new component.
+1. Απο το terminal παράθυρο, μεταβείτε στον κατάλογο που περιέχει την εφαρμογή σας.
```
```suggestion
1. Απο το terminal παράθυρο, μεταβείτε στον φάκελο που περιέχει την εφαρμογή σας.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
+* Ένα component αρχείο, `<component-name>.component.ts`
+* Ένα template αρχείο, `<component-name>.component.html`
+* Ένα CSS αρχείο, `<component-name>.component.css`
+* Ένα αρχείο προδιαγραφών δοκιμής, `<component-name>.component.spec.ts`
```
```suggestion
* Ένα test αρχείο, `<component-name>.component.spec.ts`
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
-You can change how `ng generate component` creates new components.
-For more information, see [ng generate component](cli/generate#component-command) in the Angular CLI documentation.
+Μπορείτε να αλλάξετε τον τρόπο που η εντολή `ng generate component` δημιουργεί νέα components.
+Για περισσότερες πληροφορίες, δείτε [ng generate component](cli/generate#component-command) στο εγχειρίδιο του Angular CLI.
```
```suggestion
Για περισσότερες πληροφορίες, δείτε [ng generate component](cli/generate#component-command) στην τεκμηρίωση του Angular CLI.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
 
-Although the Angular CLI is the best way to create an Angular component, you can also create a component manually.
-This section describes how to create the core component file within an existing Angular project.
+Αν και το Angular CLI είναι ο καλύτερος τρόπος για να δημιουργήσετε ένα Angular component, μπορείτε επίσης να δημιουργήσετε ένα στοιχείο με χειροκίνητο τρόπο.
```
```suggestion
Αν και το Angular CLI είναι ο καλύτερος τρόπος για να δημιουργήσετε ένα Angular component, μπορείτε επίσης να δημιουργήσετε ένα component με χειροκίνητο τρόπο.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
-Although the Angular CLI is the best way to create an Angular component, you can also create a component manually.
-This section describes how to create the core component file within an existing Angular project.
+Αν και το Angular CLI είναι ο καλύτερος τρόπος για να δημιουργήσετε ένα Angular component, μπορείτε επίσης να δημιουργήσετε ένα στοιχείο με χειροκίνητο τρόπο.
+Αυτή η ενότητα περιγράφει τον τρόπο δημιουργίας του βασικού component αρχείου σε ένα υπάρχον Angular project.
```
```suggestion
Αυτή η ενότητα περιγράφει τον τρόπο δημιουργίας του βασικού αρχείου component σε ένα υπάρχον Angular project.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
-1. Navigate to your Angular project directory.
-1. Create a new file, `<component-name>.component.ts`.
-1. At the top of the file, add the following import statement.
+1. Πλοηγηθείτε στον κατάλογο του Angular project.
```
```suggestion
1. Μεταβείτε στον φάκελο του δικού σας Angular project.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
-1. At the top of the file, add the following import statement.
+1. Πλοηγηθείτε στον κατάλογο του Angular project.
+2. Δημιουργήστε ένα νέο αρχείο, `<component-name>.component.ts`.
+3. Στην κορυφή του αρχείου, προσθέστε την άκολουθη import δήλωση.
```
```suggestion
3. Στην κορυφή του αρχείου, προσθέστε το ακόλουθο import.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
    </code-example>
 
-1. After the `import` statement, add a `@Component` decorator.
+4. Μετά την `import` δήλωση, προσθέστε έναν `@Component` decorator.
```
```suggestion
4. Μετά το `import`, προσθέστε έναν `@Component` decorator.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
    </code-example>
 
-1. Add a `class` statement that includes the code for the component.
+8.  Προσθέστε μια `class` δήλωση που περιλαμβάνει τον κώδικα για το component.
```
```suggestion
8.  Προσθέστε ένα  `class` που περιλαμβάνει τον κώδικα για το component.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -1,145 +1,146 @@
 
-A template is a block of HTML that tells Angular how to render the component in your application.
-Define a template for your component in one of two ways: by referencing an external file, or directly within the component.
+Ένα template είναι ένα μπλοκ HTML που λέει στο Angular πώς να απεικονίσει το component στην εφαρμογή σας.
```
```suggestion
Ένα template είναι ένα κομμάτι HTML που λέει στην Angular πώς να απεικονίσει το component στην εφαρμογή σας.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -148,36 +149,36 @@ For example:
 <div class="alert is-helpful">
 
-An Angular component requires a template defined using `template` or `templateUrl`. You cannot have both statements in a component.
+Ένα Angular component απαιτεί ένα template που ορίζεται με χρήση του `template` ή `templateUrl`. Δεν μπορείτε να έχετε και τις δύο δηλώσεις σε ένα component.
```
```suggestion
Ένα Angular component απαιτεί ένα template που ορίζεται με χρήση του `template` ή `templateUrl`. Δεν μπορείτε να έχετε και τις δύο ιδιότητες σε ένα component.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -148,36 +149,36 @@ For example:
 </div>
 
-## Declaring a component's styles
+## Δήλωση του στυλ ενός component
```
```suggestion
## Δήλωση των στυλ ενός component
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -148,36 +149,36 @@ For example:
+## Δήλωση του στυλ ενός component
 
-Declare component styles uses for its template in one of two ways: by referencing an external file, or directly within the component.
+Δηλώστε τις χρήσεις στυλ του component για το template του με έναν από τους δύο τρόπους: με αναφορά σε ένα εξωτερικό αρχείο ή απευθείας μέσα στο component.
```
```suggestion
Δηλώστε τα στυλ του component για το template του με έναν από τους δύο τρόπους: με αναφορά σε ένα εξωτερικό αρχείο ή απευθείας μέσα στο component.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -148,36 +149,36 @@ For example:
 </code-example>
 
-The `styles` property takes an array of strings that contain the CSS rule declarations.
+Η `styles` ιδιότητα λαμβάνει έναν πίνακα από συμβολοσειρές που περιέχουν τις δηλώσεις κανόνων CSS.
```
```suggestion
Η ιδιότητα `styles` δέχεται μια λίστα από strings που περιέχουν τους κανόνες CSS.
```

bampakoa commented on aio/content/guide/component-overview.md:
```diff
@@ -148,36 +149,36 @@ For example:
-* For more information on templates, see [Template syntax](guide/template-syntax).
+* Για μια αρχιτεκτονική επισκόπηση των components, δείτε [Εισαγωγή στα components και τα templates](guide/architecture-components).
+* Για πρόσθετες επιλογές που μπορείτε να χρησιμοποιήσετε κατά τη δημιουργία ενός component, δείτε [Component](api/core/Component) στο API Reference.
+* Για περισσότερες πληροφορίες σχετικά με τα στυλ των components, ανατρέξτε στο [Στυλ του Component](guide/component-styles).
```
```suggestion
* Για περισσότερες πληροφορίες σχετικά με τα στυλ των components, δείτε [Στυλ του Component](guide/component-styles).
```

bampakoa reviewed (changes requested):
Καταπληκτική δουλειά @HRSArgyropoulos 👏 👏 Πολύ κατανοητή η συνολική μετάφραση 🚀
